### PR TITLE
Add Bridge.NET@15.7.0 w/ git auto-update

### LINKED ISSUE #10225: [Request] Add bridge.net
**Library name:** bridge.net
**Git repository url:** https://github.com/bridgedotnet/Bridge
**npm package url(optional):** 
**License(s):** Apache 2.0
**Official homepage:** http://bridge.net/
**Wanna say something? Leave message here:**
This compiles C# user code into .js.  The output code for the user (e.g. app.js or app.min.js) needs the runtime bridge.js or bridge.min.js.  Please host bridge.js and bridge.min.js (multiple versions, if possible) on your CDN.  Thanks!

=====================
Notes from cdnjs maintainer:
You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10225
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/bridgedotnet/Bridge
 * Official website (optional, not the repository): http://bridge.net/
 * License and its reference: Apache 2.0 Ref: https://github.com/bridgedotnet/Bridge/blob/master/LICENSE.md
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 78
   - Count of stars: 801
   - Count of forks: 135

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
Add version 1.10.0 to 15.7.0 from github.com/bridgedotnet/Speed manually because
Bridge.NET provide `bridge.js` `bridge.min.js` in that repo, not from the original repo. 

@cdnjs/intern3 Please help me on reviewing this PR. Thanks.

**Comment by hare1039** [comment]
One of the dev member said he'll take a look on this PR. Let's wait for his review.

**Comment by geoffreymcgill** [comment]
If we create a **/bridge/latest** folder and add each previous .js release as a Tag, would that help?

https://github.com/bridgedotnet/Speed/tree/master/bridge/latest <-- proposed

**Comment by geoffreymcgill** [comment]
I added all the releases to the **/bridge/latest** folder and stamped each [Tag](https://github.com/bridgedotnet/Speed/tags) as I went.

https://github.com/bridgedotnet/Speed/tree/master/bridge/latest

Can the **package.json** `autoupdate` config be added?

```js
  "autoupdate": {
    "source": "git",
    "target": "git://github.com/bridgedotnet/Speed.git",
    "fileMap": [
      {
        "basePath": "bridge/latest",
        "files": [
          "*",
        ]
      }
    ]
  }
```

Hope this helps.

**Comment by hare1039** [comment]
@geoffreymcgill Add the folder would help a lot! Please add the folder. Thanks!
We can add the old version manually, so no need to worried about them.

**Comment by geoffreymcgill** [comment]
> Add the folder would help a lot! Please add the folder. 

@hare1039 – Done.

**Comment by hare1039** [comment]
@geoffreymcgill Thank you for the work! Now our bot will auto-update all the versions when this PR being merge.

@cdnjs/intern3 Please review this PR again. Thanks.

**Comment by hare1039** [comment]
@cdnjs/intern3 I fix the commit message. Please review this PR again. Thanks.

File: ajax/libs/Bridge.NET/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "author": "Bridge.NET <hello@bridge.net>",
+  "license": "Apache-2.0",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/bridgedotnet/Speed.git",
+    "fileMap": [
+      {
+        "basePath": "bridge/latest",
+        "files": [
+          "**/*"
```
Comment (sufuf3) [inline ajax/libs/Bridge.NET/package.json]: I think we don't need `bridge.d.ts`.

**Comment by hare1039** [comment]
Any `.d.ts` will ignore by `.gitignore`, so I don't explicit write here.

File: ajax/libs/Bridge.NET/package.json (hunk 1 above)
Comment (pvnr0082t) [inline ajax/libs/Bridge.NET/package.json]: We still need to exclude `*.d.ts`.

**Comment by hare1039** [comment]
@pvnr0082t OK I add the rules. Please revise this PR again. Thanks.

File: ajax/libs/Bridge.NET/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "description": "Write modern mobile and web apps in C#. Run them anywhere in JavaScript with Bridge.NET. http://bridge.net",
+  "homepage": "http://bridge.net/",
+  "keywords": [
+    "Bridge.NET",
+    "JavaScript",
+    "Compiler"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/bridgedotnet/Bridge/bridge.git"
```
Comment (pvnr0082t) [inline ajax/libs/Bridge.NET/package.json]: invalid url...

**Comment by hare1039** [comment]
@pvnr0082t Sorry I missed that. Fixed now. Please review this PR again. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I rebase the PR with master, please review this PR again. Thanks.

**Pull request merged**